dotCMS/src/main/java/com/dotmarketing/beans/MultiTree.java

```diff
@@ -44,27 +44,38 @@ public MultiTree(String htmlPage, String container, String child, String relatio
     public MultiTree() {
         this(null, null, null, LEGACY_RELATION_TYPE, 0);
     }
+    
 
+    private MultiTree(MultiTree tree) {
+        this(tree.parent1, tree.parent2, tree.child, tree.relationType, tree.treeOrder);
+    }
+    
     /** minimal constructor */
-
     public MultiTree(String htmlPage, String container, String child) {
         this(htmlPage, container, child, LEGACY_RELATION_TYPE, 0);
     }
 
     /**
-     * 
+     * @deprecated
      * {@link #getContentlet()}
      */
     @Deprecated
     public String getChild() {
         return getContentlet();
     }
 
+    /**
+     * returns the contentlet Id of the multitree
+     * @return
+     */
     public String getContentlet() {
         return this.child;
     }
 
-
+    /**
+     * @deprecated 
+     * {@link #setContentlet(String)}
+     */
     @Deprecated
     public MultiTree setChild(String child) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)